app/ui-react/syndesis/src/modules/integrations/components/editor/choice/utils.ts

```diff
@@ -20,3 +20,35 @@ export function createChoiceConfiguration(configuredProperties: {
     routingScheme,
   } as IChoiceConfiguration;
 }
+
+/**
+ * Builds a sane flow form option description from either condition expression or path, op, value properties
+ * @param option
+ */
+export function getFlowDescription(option: IFlowFormOption) {
+  if (option.condition) {
+    return option.condition;
+  }
+
+  if (option.path) {
+    return '${body.' + option.path + '} ' + option.op + " '" + option.value + "'";
```

Comment:
you could do this via an interpolated string with backticks which should make this more readable, I think it'd be like:

```
return `\${body.${option.path}} ${option.op} '${option.value}'`
```

Just need to escape the first `$` so it's printed in the resulting string.